source/Outfit.cpp

```diff
@@ -202,6 +202,7 @@ int Outfit::CanAdd(const Outfit &other, int count) const
 			count = value / -at.second + EPS;
 	}
 	
+	// TODO: return max<int>(0, count);
```

Comment:
?

Reply:
If a ship has 5 EMP Torpedo Bay launchers and 500 EMP Torpedo missiles installed and _then_ you enter the outfitter, CanAdd will return `-455`. I thought it should return `0` in that case so I left a comment to remember to get back and review that question later.